PhoneAuthUI/Sources/PhoneLogin/Models/User+Defaults.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  User+Defaults.swift
+//  teo
+//
+//  Created by Uwe Tilemann on 13.03.24.
+//
+
+import Foundation
+import SnabbleNetwork
+import SnabbleUser
+
+public extension UserDefaults {
+    // if the hosting app uses https://github.com/sindresorhus/Defaults there is a restriction:
+    // `The key name must be ASCII, not start with @, and cannot contain a dot (.).`
+    var isSignedInKey: String {
+        "io-snabble-sdk-network-user-isSignedIn"
+    }
+    func isUserSignedIn() -> Bool {
+        return bool(forKey: isSignedInKey)
+    }
+    func setUserSignedIn(_ signedIn: Bool) {
+        setValue(signedIn, forKey: isSignedInKey)
+    }
+}
+
+extension SnabbleNetwork.User {
+    func update(withDetails details: SnabbleNetwork.User.Details) {
+        print("TODO: SnabbleNetwork.User update(withDetails:)")
+//        UserDefaults.standard.setUser(.init(user: self, details: details))
+    }
+    func update(withConsent consent: SnabbleNetwork.User.Consent) {
+        print("TODO: SnabbleNetwork.User update(withConsent:)")
+//        UserDefaults.standard.setUser(.init(user: self, consent: consent))
+    }
+}
```

Comment:
Das sollten wir so nicht mergen. 

1. Die Funktion muss doch funktionieren, ansonsten wird die Informationen nicht gespeichert.

1. `standard` Defaults ist auch nicht optimal, sollen wir ein custom defaults für das SDK einführen? Damit haben wir einen eigenen Scope. Die Hosting App kann es dann auslesen.

Reply:
Ich bin mir unsicher ob es so passt, aber habe es in f1766489ce4b8cbda3dd3998a20b6bad5ba11924 angepasst. Wir müssen in VR prüfen ob das so passt und am besten integrieren wir den Branch auch direkt bei Teo, ansonsten bekommen wir es nur durch noch mehr Aufwände zusammen.